static/slides/javascript.html

```diff
@@ -432,20 +453,20 @@
              
 
                   ```javascript
-                  function humain () {
+                  function Humain () {
                    this.manger = function () {
                     console.log('Miam');
                    }
                   }
 
-                  var chloe = new humain();
-                  var marc = new humain();
+                  var chloe = new Humain();
+                  var marc = new Humain();
                   marc.manger = function () {
                       return "Ravioli";
                   }
 
                   console.log(marc.manger()); // Ravioli
-                  console.log(chloe.manger()); // Miam
+                  chloe.manger(); // Miam
```

Comment:
```suggestion
                  console.log(chloe.manger()); // Miam
```

Uniformise, ça sera plus clair